packages/client/src/utils/draftUtils.ts

```diff
@@ -27,9 +27,13 @@ import { EMPTY_STRING } from '@client/utils/constants'
 
 const getInformantEngName = (sectionData: IFormSectionData): string => {
   if (sectionData.firstNamesEng) {
-    return `${sectionData.firstNamesEng as string} ${
-      sectionData.familyNameEng as string
-    }`
+    if (sectionData.firstNamesEng && sectionData.familyNameEng) {
+      return `${sectionData.firstNamesEng as string} ${
+        sectionData.familyNameEng as string
+      }`
+    } else {
+      return `${sectionData.firstNamesEng as string}`
+    }
   } else {
     return sectionData.familyNameEng as string
```

Comment:
Would it be better if we do something like this?
```ts
  const firstName = sectionData.firstNamesEng as string | undefined
  const familyName = sectionData.familyNameEng as string | undefined
  return `${firstName ?? ''} ${familyName ?? ''}`
```
I'm saying this because either of them could be undefined right?

Reply:
ok